impl/cmd/main.go

```diff
@@ -26,29 +30,46 @@ var commitHash string
 //	@license.name	Apache 2.0
 //	@license.url	http://www.apache.org/licenses/LICENSE-2.0.html
 func main() {
-	logrus.Info("Starting up...")
+	logrus.SetFormatter(&logrus.JSONFormatter{
```

Comment:
should this be in an init block?

Reply:
It could, I don't really see a benefit to having it there but if you prefer i'm happy to move it

Reply:
in the ssi service I only did this because there was logging before the main method...I think
this seems sufficient